VideoLocker/src/main/java/org/edx/mobile/discussion/DiscussionComment.java

```diff
@@ -142,7 +142,17 @@ public String getAuthorLabel() {
                 public Date getCreatedAt() {
                     return endorsedAt;
                 }
+
+                @Override
+                public boolean isAuthorAnonymous() {
+                    return DiscussionComment.this.isAuthorAnonymous();
```

Comment:
Looking at it again, this function should always return `false`, cuz a response can't be endorsed anonymously.